### DISCUSSION TITLE
Help Needed with Deploying iPLAss Web App

### DISCUSSION THREAD

**Discussion by khanh7516** [dentsusoken/iPLAss#1556] [Q&A] [answered]
Hi,

I have developed a small application using the iPLAss platform, and now I want to deploy it.

I couldn't find any documentation specifically about deploying a web app on iPLAss. Based on the [System Architecture](https://iplass.org/en/docs/overview/index.html#_system_architecture), I believe it's possible to deploy on cloud platforms like AWS or Azure. However, I need a concrete guide on which services to use on AWS and the detailed steps to follow.

Could you provide some suggestions for deploying a web application built with iPLAss on cloud platforms like AWS? When deploying an application on AWS, is the process different from deploying a standard web application? If there are other ways to deploy the application to a live environment, I would appreciate any guidance on those as well.

Thanks so much for your help!

**Comment by KoheiKGYM** [comment, marked as answer]
Hi @khanh7516 ,
thank you for using iPLAss!

You can deploy your web application built with iPLAss as a normal Java EE (or Jakarta EE) based web application. 

Please install Tomcat 9 on a server like Amazon EC2 and deploy iPLAss web application in WAR format. You can create the WAR file of your iPLAss application using Gradle.
